cl/simple_pages/templates/v2_help/index.html

```diff
@@ -38,6 +38,7 @@ <h2 class="text-display-xs font-semibold text-greyscale-900">Getting Help on</h2
           <li><a href="{% url "donation_help" %}">Help with your donations</a></li>
           <li><a href="{% url "delete_help" %}">Help deleting your account</a></li>
           <li><a href="{% url "pray_and_pay_help" %}">Help with Pray and Pay project</a></li>
+          <li><a href="{% url "relative_dates" %}">Help with Relative Dates</a></li>
```

Comment:
Since this is related to searches, I think we should put it after the "help with advanced search parameters" item. It should not be capitalized, and maybe we can tweak it to say, "Help with relative date queries"?